en/app.ftl

```diff
@@ -616,6 +616,12 @@ banner-label-data-notification-cta = Go to Settings
 banner-label-data-notification-header-v2 = Enable new features
 banner-label-data-notification-body-cta = Learn More
 
+# Variables:
+#   $monthly_price (string) - the monthly cost (including currency symbol) for Relay Premium. Examples: $0.99, 0,99 €
```

Comment:
This comment needs to be bounded with the string containing the variable. Please move it.

Reply:
Done! Good call out